packages/editor/src/client/routes/commentList.tsx

```diff
@@ -110,7 +105,14 @@ export function CommentList() {
   const [sortField, setSortField] = useState('modifiedAt')
   const [sortOrder, setSortOrder] = useState<'asc' | 'desc'>('desc')
 
-  const [filter, setFilter] = useState('')
+  const [filter, setFilter] = useState<CommentFilter>({
```

Comment:
Since Prisma does not support index of array sorting via aggregateDocuments like mongoDB does, we have filters for the comment state here.

Reply:
does the pagination still work with that client-side implementation?